exporters/elasticsearch-exporter/src/test/java/io/zeebe/exporter/ElasticsearchClientTest.java

```diff
@@ -204,4 +204,26 @@ public void shouldFlushOnMemoryLimit() {
     // then
     assertThat(client.shouldFlush()).isTrue();
   }
+
+  @Test
+  public void shouldUseLegacyUrlIfExist() {
```

Comment:
As you can see from the other tests in the file, we tend to use a format of

```java
@Test
public void shouldTestSomething() {
  // given
  // add all the setup here, i.e. variable declaration, object instantiation, etc.
  
  // when
  // add the behaviour you want to test: for example, here, creating the client is the behaviour under test.
  
  // then
  // add your assertions here - in this case, something like the following:
  assertThat(client.client.getNodes().stream().map(Node::getHost)).containsExactly(elastic.getRestHttpHosts()[0]);
}
```

I would also argue a test with no assertions (just checking that an exception is thrown) is typically a code smell - what if we stop resolving the host at initialization and start doing so lazily? Then the test breaks. We should aim for explicit asserts as much as possible.